book/src/users/README.md

```diff
@@ -1,6 +1,6 @@
-# Users
+# Use Cases
```

Comment:
`book/src/users/README.md` and `book/src/concepts/use_cases.md` seem to be duplicate pages maybe we could hyperlink to the other one if needed instead of duplicating everything.

Reply:
Hah, yeah, just a copy that was supposed to be a move. They are identical (and one of them wasn't linked in the index), so I will delete that one.